assembly/assembly-wsmaster-war/src/main/webapp/WEB-INF/classes/che/che.properties

```diff
@@ -309,6 +309,9 @@ che.infra.kubernetes.namespace.default=<username>-che
 # Defines whether che-server should try to label the workspace namespaces.
 che.infra.kubernetes.namespace.label=true
 
+# Defines whether che-server should try to annotate the workspace namespaces.
+che.infra.kubernetes.namespace.annotation=true
```

Comment:
I would keep the `annotate` name here... It makes sense for it to be a verb in infinitive as opposed to a noun being set to `true` or `false`.

It also makes sense together with `...annotations`, e.g. `annotate` (true or false) with these `annotations`.

Reply:
ok

Reply:
It's renamed to `che.infra.kubernetes.namespace.annotate`